### PR TITLE
cake-5005 | GDPR Screen One

### PR DESCRIPTION
**Pull request by cschucode**
- Created a separate component for screen one markup
- Base styles 

Todo: 
- Add logic for Save and Continue
- Integrate Content Manager

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,6 @@
   },
   "scripts": {
     "start": "yarn clean && yarn dev",
-    "nohup": "yarn clean && nohup yarn dev &",
```
Comment (mklucsarits) [inline package.json]: Looks like you've got a bad merge here. Lots of stuff removed that shouldn't be.

File: .editorconfig [hunk 2]
```diff
@@ -1,8 +0,0 @@
-root = true
```
Comment (JoshuaRogan) [inline .editorconfig]: Why delete this file?

File: src/components/Switch-test.js [hunk 3]
```diff
@@ -1,48 +0,0 @@
-import { h, render } from 'preact';
```
Comment (JoshuaRogan) [inline src/components/Switch-test.js]: ??

**Comment by JoshuaRogan** [comment]
@cschucode I think Matt is right. Once you fix the merge issues can you please re-ping for review?

File: src/components/ScreenOne.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { h, Component } from 'preact';
+import styles from './styles.scss';
+
+class ScreenOne extends Component {
+  render({ content, text }) {
```
Comment (mklucsarits) [inline src/components/ScreenOne.js]: Indentation is off in this file. It should be 4 spaces.

File: package.json (hunk 1 above)
Comment (cschucode) [reply package.json]: Added this back.

File: .editorconfig (hunk 2 above)
Comment (cschucode) [reply .editorconfig]: Added this back too.

File: src/components/styles.scss [hunk 5]
```diff
@@ -147,87 +138,81 @@ $color-text: #002a32;
...
-  p:last-of-type {
-    margin-bottom: 0; // save some space to prevent scroll bars
+  &.desktop {
+      @media #{$mobile-devices} {
+        display: none;
+      }
   }
 }
 
-.switch {
```
Comment (vforge) [inline src/components/styles.scss]: This shouldn't be deleted

File: selenium/jenkins [hunk 6]
```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent {
         node {
-            label 'docker-daemon'
```
Comment (vforge) [inline selenium/jenkins]: I don't think we want to change this in this PR

File: src/components/ScreenOne.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import { h, Component } from 'preact';
+import styles from './styles.scss';
```
Comment (vforge) [inline src/components/ScreenOne.js]: Please separate styles for ScreenOne.

File: webpack.config.js [hunk 8]
```diff
@@ -23,7 +23,6 @@ if (process.env.NODE_ENV === 'development') {
     topLevelOptions = {
         serve: {
             port: 3000,
-            host: '0.0.0.0',
```
Comment (vforge) [inline webpack.config.js]: Bad merge leftover

File: selenium/jenkins (hunk 6 above)
Comment (cschucode) [reply selenium/jenkins]: This was a part of the previous pr that @frankfarmer recommended.

**Pull request closed without merging**